src/help/help_browser.cpp

```diff
@@ -168,6 +168,27 @@ void help_browser::handle_event(const SDL_Event &event)
 				}
 			}
 		}
+		else {
+			// ERR_NG <<"MB:" << (int) mouse_event.button << "\n"; 			
+			bool mouse_back = false;
```

Comment:
You can simplify the assignment here:
```
const bool mouse_back = !back_button_.hidden() && mouse_event.button == SDL_BUTTON_X1;
```
Same for `mouse_forward`.

Reply:
Yeah. I made a new push 2b2f434 . Is there anything else need to do with git operation?